tools/src/test/resources/com/nedap/archie/serializer/adl/openEHR-TEST_PKG-WHOLE.primitive_types.v1.adls

```diff
@@ -158,11 +158,15 @@ definition
 		duration_attr34 matches {PTs/|P38W..P39W4D|}
 		duration_attr35 matches {PThm/|P38W..P39W4D|}
 		duration_attr36 matches {|>=PT0S|}
+        duration_attr37 matches {|>=-PT10S|}
+        duration_attr38 matches {|>=-P10D|}
 		duration_attr104 matches {PD; P4D}
+		duration_attr105 matches {PD; -P4D}
 		duration_attr118 matches {|<=PT1H|; PT30M}
 		duration_attr128 matches {Pw/|P38W..P39W4D|; P38W2D}
+		duration_attr129 matches {Pw/|-P38W..P39W4D|; -P36W2D}
 		duration_attr118 matches {|<=PT1H|; PT30M}
-		duration_attr128 matches {Pw/|P38W..P39W4D|; P38W2D}
+		duration_attr130 matches {Pw/|P38W..P39W4D|; P38W2D}
```

Comment:
`duration_attr130` is the same as `duration_attr128`, `duration_attr118` is also defined twice.